documentation/jasypt.asciidoc

```diff
@@ -0,0 +1,76 @@
+:toc:
```

Comment:
Same as in other PRs. Sorry for the confusion. We discussed this in the team some time ago but nobody told this to the new people that joined the team recently:
We decided not to add redundant AsciiDoc documentation for commandlet usage since we have a very sophisticated help now that can give you all the information you need.
The redundant AsciiDoc will only outdate sooner or later when somebody changes a commandlet but forgets to update the AsciiDoc.
We already had this a lot in devonfw-ide.
See #247